src/index.js

```diff
@@ -0,0 +1,16 @@
+import * as Icons from './assets/icons';
+export { Icons };
```

Comment:
so do you suggest using this or what's in line 1?

Reply:
It seems that using:
```
export * as Icons from './assets/icons';
```
Is experimental and not supported.
The solution is to split it to two:
```
import * as Icons from './assets/icons';
export { Icons };
```

Reply:
@shlomitc was this what you meant?